project.clj

```diff
@@ -4,74 +4,40 @@
   :license {:name "Eclipse Public License"
             :url "http://www.eclipse.org/legal/epl-v10.html"}
 
-  :dependencies [[org.clojure/test.check "0.9.0"]]
+  :dependencies [[org.clojure/test.check "0.9.0"]
+                 [prismatic/schema "1.0.5"]]
 
   :profiles {:dev {:dependencies [[org.clojure/clojure "1.8.0"]
-                                  [org.clojure/clojurescript "1.7.228"]
-                                  [org.clojure/tools.nrepl "0.2.5"]
-                                  [prismatic/schema "1.0.5"]]
-                   :plugins [[com.keminglabs/cljx "0.6.0" :exclusions [org.clojure/clojure]]
-                             [codox "0.8.8"]
-                             [lein-cljsbuild "1.0.5"]
-                             [com.cemerick/clojurescript.test "0.3.1"]]
-                   :cljx {:builds [{:source-paths ["src"]
-                                    :output-path "target/generated/src/clj"
-                                    :rules :clj}
-                                   {:source-paths ["src"]
-                                    :output-path "target/generated/src/cljs"
-                                    :rules :cljs}
-                                   {:source-paths ["test"]
-                                    :output-path "target/generated/test/clj"
-                                    :rules :clj}
-                                   {:source-paths ["test"]
-                                    :output-path "target/generated/test/cljs"
-                                    :rules :cljs}]}}
+                                  [org.clojure/clojurescript "1.7.228"]]
+                   :plugins [[lein-codox "0.9.4"]]}
              :1.7 {:dependencies [[org.clojure/clojure "1.7.0"]]}}
 
   :aliases {"all" ["with-profile" "dev:dev,1.7"]
-            "deploy" ["do" "clean," "cljx" "once," "deploy" "clojars"]
-            "test" ["do" "clean," "cljx" "once," "test," "with-profile" "dev" "cljsbuild" "test"]}
```

Comment:
Done! We'll need to write a script for getting the phantom dep if we want to do CI down the line.